lib/ReactViews/Custom/Chart/BottomDockChart.tsx

```diff
@@ -322,28 +330,31 @@ class Chart extends React.Component {
             </Group>
           </svg>
           <Tooltip {...this.tooltip} />
-          <PointsOnMap terria={terria} chartItems={this.chartItems} />
+          {this.chartItems.map((chartItem: ChartItem) =>
```

Comment:
Had to change this one completely compared to what was there before.

I followed the pattern of mapping over things from line 292, and then made `pointOnMap` return null when there wasn't a point on map. I think the old code is broken, but I have no idea if what I've done is any better.